keps/sig-cloud-provider/2436-controller-manager-leader-migration/README.md

```diff
@@ -350,7 +355,8 @@ disabled.
 
 ###### How can a rollout or rollback fail? Can it impact already running workloads?
 
-The rollout may fail if the configuration file does not represent correct controller-to-manager.
+The rollout may fail if the configuration file does not represent correct controller-to-manager assignment
```

Comment:
Ideally we should force the misconfigured KCM or CCM to not come up. I realize that may not always be possible.